src/dvclive/lightning.py

```diff
@@ -17,11 +17,16 @@ def __init__(
         dir: Optional[str] = None,  # noqa pylint: disable=redefined-builtin
         resume: bool = False,
         report: Optional[str] = "auto",
+        save_dvc_exp: bool = False,
```

Comment:
@dberenbaum named the option like this. wdyt?

Reply:
I don't know if it makes sense for the `dvc.yaml` saving part and whether those should be coupled together, but I think it's fine for now.

Reply:
I though your concerns also applied to both the git ref and the `dvc.yaml` creation